Feature/drop table in ui

kaspersorensen opened the pull request:
Made it possible to right click a table and drop it from the UI.
Also fixed a bad layout issue with the 'Create staging table' property panes.

LosD commented on desktop/ui/src/main/java/org/datacleaner/widgets/tree/TableMouseListener.java:
```diff
@@ -158,6 +162,24 @@ public void actionPerformed(ActionEvent e) {
+                    });
+                    popup.add(dropTableMenuItem);
+                }
+
```
If possible, it would be nice to get some of this code out of this event handler. It is already waaaaayyyy too complex.

LosD commented on engine/core/src/main/java/org/datacleaner/util/SchemaComparator.java:
```diff
@@ -53,7 +54,7 @@ public static boolean isInformationSchema(Schema schema) {
 			return false;
 		}
-		return "information_schema".equals(name.toLowerCase());
+		return MetaModelHelper.isInformationSchema(schema);
```
MetaModelHelper already does the name check and lowercasing, so no need to repeat it.

LosD commented:
A few very minor things, but overall looks good

kaspersorensen commented:
Fixed the review remarks. Updated the icons too:
![image](https://cloud.githubusercontent.com/assets/291450/8182711/5b8d3686-1432-11e5-9856-5b7db5d67002.png)
